dnsext-dnssec/dnsext-dnssec.cabal

```diff
@@ -45,6 +47,7 @@ library
         dnsext-types,
         bytestring,
         containers,
+        base32,
```

Comment:
With GHC 9.4.3, `base32` (which depends on text) cannot be resolved.
Would you try GHC 9.4.3 by yourself?
One alternative is using `base32-bytestring`, I guess.

Reply:
Depending `text` is a bad idea, anyway.

Reply:
`base32-bytestring` has build errors

```
Building library for base32-bytestring-0.2.1.0..
[1 of 3] Compiling Data.ByteString.Base32.Internal ( src/Data/ByteString/Base32/Internal.hs, dist/build/Data/ByteString/Base32/Internal.o, dist/build/Data/ByteString/Base32/Internal.dyn_o )

src/Data/ByteString/Base32/Internal.hs:73:1: error:
    Parse error in pattern: unpack5Ptr In a function binding for the ‘@’ operator.
                            Perhaps you meant an as-pattern, which must not be surrounded by whitespace
   |
73 | unpack5Ptr !tbl bs @ (PS fptr off sz) =
   | ^^^^^^^^^^^^^^^^^^
```

Reply:
I found the encoder in dnsext-types/DNS/Types/Base32Hex.hs.
What do you think about adding a decoder there?

Reply:
Go ahead.

Reply:
After merging #50, rebase this PR and add the fixes